t/data/openqa/share/tests/opensuse/tests/openQA/kate.pm

```diff
@@ -0,0 +1,20 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Package: kate
+# Summary: Test the KDE text editor can be installed, started, typing works
+#   and closed
+# Maintainer: Oliver Kurz <okurz@suse.de>
+
```

Comment:
I would avoid the huge comment.

Reply:
I guess it's fine as an example

Reply:
It doesn't seem like the test covers this? If the metadata is searched, I would welcome it

Reply:
is used by `t/ui/10-test_overview.t`